tests/publiccloud/run_ltp.pm

```diff
@@ -130,8 +130,8 @@ sub run {
         }
     }
 
-    my $runltp_ng_repo = get_var("LTP_RUN_NG_REPO", "https://github.com/metan-ucw/runltp-ng.git");
-    my $runltp_ng_branch = get_var("LTP_RUN_NG_BRANCH", "master");
+    my $runltp_ng_repo = get_var("LTP_RUN_NG_REPO", "https://github.com/acerv/runltp-ng.git");
+    my $runltp_ng_branch = get_var("LTP_RUN_NG_BRANCH", "ssh");
```

Comment:
this raise a warning for me , why we not using master ? is ssh feature not fully ready to use ? I think I would need to talk about that with kernel-qa before fully switching to this approach ...

Reply:
@pevik , @acerv can you please comment on this

Reply:
@asmorodskyi the feature is under development for a main reason: runltp-ng will be part of the LTP upstream. That means we are going to test a simple version, plain python 3.6+ without external dependency. When things are getting more stable and fixed, SSH support will be introduced. For now, SSH stays outside master to wait for LTP upstream test